delta/plugins/composite-views/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/compositeviews/CompositeViews.scala

```diff
@@ -534,14 +533,9 @@ object CompositeViews {
       case Some(s) if s.deprecated   =>
         IO.raiseError(ViewIsDeprecated(c.id))
       case Some(s)                   =>
+        val newRev = s.rev + 1
```

Comment:
When `newRev` is used here or in further methods I have to say I keep losing track of what kind of revision we are dealing with

Reply:
Yep, I try to address that